### PR TITLE
Added SSL doc under "Administering" Topic

### PR DESCRIPTION
**Pull request by eyedarts**

### CODE REVIEW

**Comment by garply** [comment]
Hey admins! Can one of you let me know if it's safe to build this pull request? See https://alpha.sandstorm.io/grain/kB7bofATTL2jGuKSFwmqdA/GithubPRCommands for a list of commands.

File: mkdocs.yml [hunk 1]
```diff
@@ -39,4 +39,5 @@ pages:
     - "Backups": "administering/backups.md"
     - "Wildcard hosts": "administering/wildcard.md"
     - "Reverse Proxy": "administering/reverse-proxy.md"
+    - "SSL": "administering/ssl.md"
```
Comment (paulproteus) [inline mkdocs.yml]: Can I suggest you title this "Self-signed SSL" rather than just "SSL"?

Comment (eyedarts) [inline mkdocs.yml]: Done.

**Comment by paulproteus** [comment]
Happy to merge this! Thanks so much for writing this.

**Comment by paulproteus** [comment]
https://docs.sandstorm.io/en/latest/administering/ssl/ contains this doc now. Again, thanks @eyedarts for writing this!

**Pull request merged**